__tests__/unit/components/App/AppUpdater.spec.js

```diff
@@ -0,0 +1,199 @@
+import { shallowMount, createLocalVue } from '@vue/test-utils'
+import Vuex from 'vuex'
+import { AppUpdater } from '@/components/App'
+
+const localVue = createLocalVue()
+localVue.use(Vuex)
+
+jest.mock('electron', () => ({
+  ipcRenderer: {
+    on: jest.fn(),
+    send: jest.fn()
+  }
+}))
+
+let wrapper
+
+const availableRelease = () => {
+  return {
+    version: '0.1',
+    releaseNotes: '<div><h3>Title</h3><a href="#">Test Link</a></div>'
+  }
+}
+
+const store = new Vuex.Store({
+  modules: {
+    updater: {
+      namespaced: true,
+      getters: {
+        availableRelease
+      }
+    }
+  }
+})
+
+const mockData = {
+  $t: () => 'Mock Translaton',
+  process: {
+    platform: 'linux'
+  },
+  assets_loadImage: data => data
+}
+
+const createWrapper = (mocks = mockData) => {
+  wrapper = shallowMount(AppUpdater, {
+    store,
+    mocks
+  })
+}
+
+describe('AppUpdater', () => {
+  beforeEach(() => {
+    createWrapper()
+  })
+
+  it('should render', () => {
+    expect(wrapper.isVueInstance()).toBeTruthy()
+    expect(wrapper.contains('.AppUpdater')).toBeTruthy()
+  })
+
+  it('should format percentage', () => {
+    expect(wrapper.vm.formattedPercentage).toBe('0.00%')
+    wrapper.vm.progressUpdate.percent = 5
+    expect(wrapper.vm.formattedPercentage).toBe('5.00%')
+    wrapper.vm.progressUpdate.percent = 5.56567688
+    expect(wrapper.vm.formattedPercentage).toBe('5.57%')
+  })
+
+  it('should return if Linux', () => {
+    const originalPlatform = process.platform
+
+    Object.defineProperty(process, 'platform', {
+      value: 'win95'
+    })
+    createWrapper()
+    expect(wrapper.vm.isLinux).toBe(false)
+
+    Object.defineProperty(process, 'platform', {
+      value: 'linux'
+    })
+    createWrapper()
+    expect(wrapper.vm.isLinux).toBe(true)
+
+    Object.defineProperty(process, 'platform', {
+      value: originalPlatform
+    })
+  })
+
+  it('should return isAppImage', () => {
+    const originalEnv = process.env
+
+    Object.defineProperty(process, 'env', {
+      value: { APPIMAGE: 'a value' }
+    })
+    createWrapper()
+    expect(wrapper.vm.isAppImage).toBe(true)
+
+    Object.defineProperty(process, 'env', {
+      value: { APPIMAGE: null }
+    })
+    createWrapper()
+    expect(wrapper.vm.isAppImage).toBe(false)
+
+    Object.defineProperty(process, 'env', {
+      value: originalEnv
+    })
+  })
+
+  it('should return title if not authorized downloading', () => {
```

Comment:
Change from:

`it('should return title if not authorized downloading', () => {`

To:

`it('should return the title if download is not authorized', () => {`